cypress/integration/homepage_spec.js

```diff
@@ -71,6 +71,47 @@ describe('Homepage', () => {
         .url().should('not.equal', 'http://localhost:3000/')
     })
 
+    it('should change the number of posters when search match', () => {
+      cy.get('.poster-image')
+        .should('have.length', 10) 
+        .get('input[type=search]')
+        .should('be.visible')
+        .type('mulan')
+        .get('.poster-image')
+        .should('have.length', 1) 
+        .focused().clear()
+        .get('input[type=search]')
+        .type('m')
+        .get('.poster-image')
+        .should('have.length', 4) 
+    })
+
+    it('should update title of the page accordingly', () => {
```

Comment:
Thanks for being thorough with testing to make sure the user is seeing the right title.